src/components/NotificationBar/index.tsx

```diff
@@ -47,26 +46,36 @@ const SMenuItem = styled(MenuItem)``;
 export default function NotificationBar() {
   const dispatch = useAppDispatch();
   const navigate = useNavigate();
+  const { notifications } = useAppSelector((state) => state.app);
   const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
   const open = Boolean(anchorEl);
+
+  useEffect(() => {
+    let clearNotificationsTimer = setTimeout(
+      () => dispatch(appActions.clearExpiredNotifications()),
+      2000
+    );
+    return () => {
+      clearTimeout(clearNotificationsTimer);
+    };
+  });
```

Comment:
let's not hide the notifications for now, we still have no mechanism of making sure that users have seen them